tests/agents/test_max_height.py

```diff
@@ -0,0 +1,97 @@
+import holodeck
+import uuid
+from copy import deepcopy
+
+from tests.utils.equality import almost_equal
+
+uav_config = {
+    "name": "test_max_height",
+    "world": "TestWorld",
+    "main_agent": "uav0",
+    "agents": [
+        {
+            "agent_name": "uav0",
+            "agent_type": "UavAgent",
+            "sensors": [
+                {
+                    "sensor_type": "LocationSensor",
+                }
+            ],
+            "control_scheme": 0,
+            "location": [0, 0, 2],
+            "max_height": 3
+        }
+    ]
+}
+
+binary_path = holodeck.packagemanager.get_binary_path_for_package("DefaultWorlds")
+
+def test_stuck_at_mh():
```

Comment:
Could we just do `test_stuck_at_max_height` (and the same thing for `test_not_stuck_at_mh`)? I'm just thinking about whether I could understand what this test is doing without the context of the surrounding file.